deploy/osd-14634-disable-cpms/10-osd-disable-cpms.CronJob.yaml

```
@@ -0,0 +1,39 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: osd-disable-cpms
+  namespace: openshift-machine-api
+spec:
+  failedJobsHistoryLimit: 3
+  successfulJobsHistoryLimit: 1
+  concurrencyPolicy: Replace
+  schedule: "0 */1 * * *"
+  jobTemplate:
+    spec:
+      ttlSecondsAfterFinished: 180
+      template:
+        spec:
+          affinity:
+            nodeAffinity:
+              preferredDuringSchedulingIgnoredDuringExecution:
+              - preference:
+                  matchExpressions:
+                  - key: node-role.kubernetes.io/infra
+                    operator: Exists
+                weight: 1
+          tolerations:
+            - effect: NoSchedule
+              key: node-role.kubernetes.io/infra
+              operator: Exists
+          serviceAccountName: osd-disable-cpms
+          restartPolicy: Never
+          containers:
+          - name: osd-disable-cpms
+            image: image-registry.openshift-image-registry.svc:5000/openshift/cli:latest
+            imagePullPolicy: Always
+            args:
+            - /bin/bash
+            - -c
+            - |
+              # the appropriate way to disable CPMS is to delete the CPMS CR.
+              oc delete controlplanemachineset cluster -n openshift-machine-api
```

Can we fully qualify the resource here?
```suggestion
              oc delete controlplanemachineset.machine.openshift.io/cluster -n openshift-machine-api
```

---

Done.